lua/acf/menu/items_cl/baseplates.lua

```diff
@@ -0,0 +1,70 @@
+local ACF        = ACF
+
+local gridMaterial = CreateMaterial("acf_bp_vis_spropgrid1", "VertexLitGeneric", {
+	["$basetexture"] = "hunter/myplastic",
+	["$model"] = 1,
+	["$translucent"] = 1,
+	["$vertexalpha"] = 1,
+	["$vertexcolor"] = 1
+})
+
+local function CreateMenu(Menu)
+	ACF.SetToolMode("acf_menu", "Spawner", "Baseplate")
+	ACF.SetClientData("PrimaryClass", "acf_baseplate")
+	ACF.SetClientData("SecondaryClass", "N/A")
+
+	Menu:AddTitle("Baseplate Settings")
+
+	Menu:AddLabel("The root entity of all ACF contraptions.")
+	local SizeX     = Menu:AddSlider("Plate Width (gmu)", 36, 96, 2)
+	local SizeY     = Menu:AddSlider("Plate Length (gmu)", 36, 420, 2)
+	local SizeZ     = Menu:AddSlider("Plate Thickness (gmu)", 0.5, 3, 2)
+
+	Menu:AddLabel("Comparing the current dimensions with a 105mm Howitzer:")
```

Comment:
Could we put all of this stuff within a Baseplate Information dropdown just for the sake of visual consistency with other menus?